packages/styles/scss/patterns/sections/leadspace/_leadspace.scss

```diff
@@ -72,7 +72,6 @@ $btn-min-width: 26;
   .#{$prefix}--leadspace {
     margin-right: auto;
     margin-left: auto;
-    max-width: 99rem;
```

Comment:
@andysherman2121 wasn't there a change you made that added this recently?

cc: @RaphaelAmadeu

Reply:
Yeah the leadspace is supposed to have a max width so that when it reaches 99rem's. It has white gutters on either side of it

Reply:
@larahanlon2 can you weigh in on the max width treatment of the leadspace?

Reply:
@andysherman2121 are we not using the grid handle that `max-width`?

Reply:
@andysherman2121 is correct in saying that the lead space needs to max out with the grid – just like the masthead. Here's the visual spec that we confirmed before:  
<img width="1618" alt="Screen Shot 2020-04-01 at 9 07 43 AM" src="https://user-images.githubusercontent.com/53093550/78142614-1edce500-73fb-11ea-9072-fa90c5955402.png">

Reply:
@larahanlon2 @kennylam does the masthead/footer also max out at that width? Should we be maxing the width on the dotcom shell level instead?

Reply:
The **masthead** has the same max behavior except that the HR at the bottom of the L0 continues to span full width. I believe the content in the **footer** also does the same except the background color continues to span the width of the browser. All of this is to ensure that the content (text, components etc.) are aligned from pattern to pattern.

Reply:
@photodow we're not using the grid to handle the max width. The max width is set at the max width breakpoint

Reply:
> @larahanlon2 @kennylam does the masthead/footer also max out at that width? Should we be maxing the width on the dotcom shell level instead?

@jeffchew we're already doing that. All content minus masthead/footer in `Dotcomshell` sits inside `bx--grid`, which maxes at 99rem. I didn't get a chance to familiarize myself with `leadspace` yet. Not sure why we need to specify a max width for that, instead of 100% of the parent.

![image](https://user-images.githubusercontent.com/909118/78178212-464ca580-742d-11ea-941c-e05030f3d444.png)

Reply:
@RaphaelAmadeu @kennylam @andysherman2121 So where did we land on this, we're removing the max-width for the leadspace? I'm assuming it still maxes out at 99rems if it's sitting in the DotcomShell, right?

Reply:
I actually don't see a max-width happening for the dotcom shell:

![Screen Shot 2020-04-03 at 8 39 39 AM](https://user-images.githubusercontent.com/1641214/78361722-da7e5000-7586-11ea-9027-6f2393eb58e8.png)

Perhaps this PR should also add the max-width to the `bx--dotcom-shell`?

Reply:
Resolving this, we know we need to remove max-width for the leadspace, plus @RaphaelAmadeu just added the grid wrapper to storybook.